Makefile

```diff
@@ -47,6 +48,9 @@ proto:
 # TODO(meling): Split the proto target to avoid generating too new typescript... Need to fix #1147 first; after which we should merge this target with the proto target.
 proto-ui: $(protopatch)
 	buf generate --template buf.gen.ui.yaml --exclude-path patch
+	@echo "Removing protopatch imports from $(proto_ts)"
+	@perl -i -ne "print unless /import\s*\{\s*file_patch_go\s*\}\s*from\s*[\"'].*patch\/go_pb[\"'];/" $(proto_ts)
```

Comment:
@JosteinLindhom I keep forgetting that `sed` is not a great choice for cross-platform commands because macOS sed and GNU sed is interpreting the `-i` option differently. Does WSL have `perl` installed by default?